pyroute2/protocols/__init__.py

```diff
@@ -118,9 +118,12 @@ def _encode_mac(value: str) -> list[int]:
 
 
 class msg(dict):
-    buf = None
+    # FIXME: figure out why these are declared as class attributes
```

Comment:
@svinota could you tell me why there seem to be instance variable declared as class attributes in some places ?

Reply:
No particular reason. Just remnants of prototyping.